frontend/public/components/utils/managed-by.tsx

```diff
@@ -59,12 +59,14 @@ export const ManagedByOperatorLink: React.SFC<ManagedByLinkProps> = (props) => {
   if (owner && csv) {
     return (
       <div className={classNames('co-m-pane__heading-owner', className)}>
-        Managed by{' '}
-        <ManagedByOperatorResourceLink
-          namespace={namespace}
-          csvName={csv.metadata.name}
-          owner={owner}
-        />
+        <span className="co-resource-item">
+          Managed by{' '}
+          <ManagedByOperatorResourceLink
+            namespace={namespace}
+            csvName={csv.metadata.name}
+            owner={owner}
+          />
+        </span>
```

Comment:
Better to add an optional `className` prop to the component and send it from here.
```suggestion
        Managed by{' '}
          <ManagedByOperatorResourceLink
            className="co-resource-item"
            namespace={namespace}
            csvName={csv.metadata.name}
            owner={owner}
          />
```

Reply:
Updated.